src/lambdas/assets-deployment/patch-fetch.js

```diff
@@ -47,15 +66,27 @@ window.fetch = async function patchedFetch(input, init) {
   }
 
   const body = init.body;
+  let bodyContent;
```

Comment:
nit: This is getting a bit complex.
Might be easier to read if extracted out to a method.
Then this can be immutable etc.

```
const bodyContent = this.getBodyConent(init.body)
```

Would also allow you to simplify the nesting with a quick return:

```javascript
function getBodyContent(body) {
  if (!body) {
    return "";
  }

  // Rest of happy path logic at single nesting level
}
```

Reply:
For `instanceof FormData` condition, I actually need acccess to `headers` within patchedFetch function scope so I'm going to leave code architecture as-is, please make PR if you think there is better way.